internal/transport/transporter.go

```diff
@@ -1,6 +1,8 @@
 package transport
 
-import "crypto/tls"
+import (
```

Comment:
This is an unnecessary formatting change. `go fmt` probably doesn't catch this because it's an acceptable format, but it's unncessary change considering we aren't adding or removing a package import here.